SCICompanionLib/Src/Dialogs/GamePropertiesDialog.cpp

```diff
@@ -56,13 +56,18 @@ void CGamePropertiesDialog::DoDataExchange(CDataExchange* pDX)
 
 	DDX_Control(pDX, IDC_STATIC4, m_wndStatic4);
 	DDX_Control(pDX, IDC_COMBOLANGUAGE, m_wndComboLanguage);
+
 //	LangSyntax lang = appState->GetResourceMap().Helper().GetDefaultGameLanguage();
 //	m_wndComboLanguage.SetCurSel((int)lang);
 //#ifdef DISABLE_STUDIO
 //	m_wndComboLanguage.EnableWindow(FALSE);
 //#endif
-	m_wndComboLanguage.SetCurSel(appState->GetResourceMap().Helper().GetCodepage() == 1252 ? 1 : 0);
 
+	// Change for accepting Codepage 850
+	// m_wndComboLanguage.SetCurSel(appState->GetResourceMap().Helper().GetCodepage() == 1252 ? 2 : 1);
+	m_wndComboLanguage.SetCurSel(appState->GetResourceMap().Helper().GetCodepage() == 1252 ? 2 :
```

Comment:
Would highly suggest using a temporary variable to hold the value returned by `GetCodePage` and using that instead of calling the function twice.